sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -58,12 +57,13 @@ export function createSpanFunction({ packagePrefix, namespace }: SpanConfig) {
     const newTracingOptions: OperationTracingOptions = {
       ...tracingOptions,
       spanOptions: newSpanOptions
+      // TODO: .context soon.
```

Comment:
If we're keeping this TODO, it may be a good idea to have an issue tracking it

Reply:
Yeah, that's a good point. It's the literal next piece of work once the new opentelemetry library is referenced (so it will definitely not be forgotten!)

Reply:
(Basically, this issue is #13573)